### ISSUE TITLE
Confused

### ISSUE THREAD

**Issue by bhanu7755** [greengerong/Prerender_asp_mvc#22]
Hi,

I built an application using AngularJS and Web API and hosted in IIS . I want to make my application SEO friendly. To do this i have done all the below steps

1. Install Nuget Package Prerender.io
2. Added meta tag in index.html
3. Added  $locationProvider.hashPrefix('!'); in the config file.
4. Updated my token in Web.Config file  <prerender token="mytoken"></prerender>  

But, still I'm seeing 
"You need to install your Prerender token before you can start caching your pages."
And unable to cache my pages.

Could you please help me.

**Comment by thoop** [comment]
Can you send an email to support@prerender.io with some more information about your current middleware setup? We'd be happy to look into this more with you.

**Comment by bhanu7755** [comment]
Thanks for reply. Just did..

**Comment by Kriyainc** [comment]
Hi Bhanu - Were you able to get this resolved? Have the exact same set up, Angular, Web API on IIS and looking for a solution...

**Comment by thoop** [comment]
@Kriyainc Happy to help you get set up. Would you email us at support@prerender.io? Send over your config and your website so that we can test and see if it's working correctly.

**Comment by Kriyainc** [comment]
Thanks for your response...I will send over the details you mentioned to support@prerender.io

**Comment by Kriyainc** [comment]
Hello - Sorry about the delay. Our project is ASP.Net website, where as this Prerender solution is for ASP.Net MVC project. The difference as you maybe aware is that, ASP.Net website project does not have App_Start and AssemblyInfo.cs. We are trying to add these manually to our project in order to add Pre-render to the project. Should be done very shortly, and will let u know if it worked...Thanks

**Comment by greengerong** [comment]
Yes. You can reuse all code, but use asp.net interceptor. And do not use App_Start to register.

**Comment by bobrinch** [comment]
I also use ASP.NET but without the MVC part. Can you please put some more words on your one line:

_"Yes. You can reuse all code, but use asp.net interceptor. And do not use App_Start to register."_

Thanks :-)
